src/queue-list/laboratory-queue.scss

```diff
@@ -210,4 +210,9 @@ title {
   @include type.type-style('heading-compact-02');
   color: $text-02;
   margin-bottom: 0.5rem;
-}
\ No newline at end of file
+}
+
+  .table {
```

Comment:
Let's change this to something more intentional and also not to conflict with other usages. What do u think of changing it to `.single-line-display`